event-handlers/src/main/java/no/sikt/nva/nvi/events/persist/UpsertNviCandidateHandler.java

```diff
@@ -36,22 +35,18 @@ public class UpsertNviCandidateHandler implements RequestHandler<SQSEvent, Void>
     private static final String UPSERT_CANDIDATE_DLQ_QUEUE_URL = "UPSERT_CANDIDATE_DLQ_QUEUE_URL";
     private static final String UPSERT_CANDIDATE_FAILED_MESSAGE = "Failed to upsert candidate for publication: {}";
     private final CandidateRepository repository;
-    private final PeriodRepository periodRepository;
-
     private final QueueClient queueClient;
     private final String dlqUrl;
 
     @JacocoGenerated
     public UpsertNviCandidateHandler() {
```

Comment:
Not related to this PR, but:
Do we actually need this constructor? It isn't used and the tests still pass if I remove it.

Reply:
Yes, it's the default constructor used to initialize the Lambda in AWS